pytket/docs/passes.rst

```diff
@@ -10,3 +10,9 @@ pytket.passes.script
 
 .. automodule:: pytket.passes.script
     :members: compilation_pass_from_script, compilation_pass_grammar
+
```

Comment:
This PR seems to include commits not relevant to it; could you please rebase your branch to `develop` (or make a new one) and push again? Thanks!